src/constants.ts

```diff
@@ -10,9 +10,24 @@ export const API_PATH = `/api/v${API_VERSION}`;
 
 export const DEFAULT_ZONE = ethers.constants.AddressZero;
 
+// Ignore eslint no-unused-modules for below to keep backward compatibility
+// in case a downstream user was already using these imports directly.
+// These can be made non-exported in next major-versioned release.
+// eslint-disable-next-line import/no-unused-modules
 export const SHARED_STOREFRONT_ADDRESS_MAINNET =
   "0x495f947276749ce646f68ac8c248420045cb7b5e";
+// eslint-disable-next-line import/no-unused-modules
 export const SHARED_STOREFRONT_ADDRESS_GOERLI =
   "0x804159144aefb1dc17b171afcefa5b33746c722f";
+const SHARED_STOREFRONT_ADDRESS_POLYGON =
+  "0x2953399124f0cbb46d2cbacd8a89cf0599974963";
+const SHARED_STOREFRONT_ADDRESS_KLAYTN =
+  "0x5bc519d852f7ca2c8cf2d095299d5bb2d13f02c9";
+export const SHARED_STOREFRONT_ADDRESSES = [
+  SHARED_STOREFRONT_ADDRESS_MAINNET,
+  SHARED_STOREFRONT_ADDRESS_GOERLI,
+  SHARED_STOREFRONT_ADDRESS_POLYGON,
+  SHARED_STOREFRONT_ADDRESS_KLAYTN,
+].map((address) => address.toLowerCase());
```

Comment:
any particular reason for moving from set to array here? would think the uniqueness properties of a set would be nice -- but maybe im just thinking in python

Reply:
yeah, just felt unneeded really, you are technically correct but since we are defining it ourselves here with no user input i feel like it's no biggie

Reply:
sounds good!